test/collections/Vector2DTest.ooc

```diff
@@ -20,30 +20,107 @@ use ooc-collections
 
 Vector2DTest: class extends Fixture {
 	init: func {
+		sizeXS := 3
+		sizeS := 5
+		sizeM := 10
+		sizeL := 15
+		sizeXL := 20
+		vector2D := Vector2D<Int> new(sizeM, sizeM)
 		super("Vector2D")
-		this add("vector2d list cover create", func {
-			vector2D := Vector2D<Int> new(10, 10)
-			expect(vector2D rowCapacity, is equal to(10))
-			expect(vector2D columnCapacity, is equal to(10))
-
-			for (i in 0 .. 10)
-				for (j in 0 .. 10)
+		this add("capacity", func {
+			expect(vector2D rowCapacity, is equal to(sizeM))
+			expect(vector2D columnCapacity, is equal to(sizeM))
+		})
+		this add("get/set operators", func {
+			for (i in 0 .. sizeM)
+				for (j in 0 .. sizeM)
```

Comment:
I should have told you this before, but, if you want to iterate over all elements of `vector2D`, why not use `vector2D rowCapacity` and `vector2D columnCapacity` instead of `sizeM` as the loop bound?